src/imguiclient/imguiapp.cpp

```diff
@@ -1492,8 +1497,11 @@ void ImGuiApp::ComparisonManagerWindows()
     for (size_t i = 0; i < count; ++i)
     {
         ProgramComparisonDescriptor &descriptor = *m_programComparisons[i];
+        const std::string title = fmt::format("Assembler Comparison {:d}", descriptor.m_id);
 
-        if (descriptor.m_imguiComparisonWindowOpened)
+        ImScoped::Window window(title.c_str(), &descriptor.m_imguiComparisonWindowOpened);
+        ImScoped::ID id(i);
+        if (window.IsContentVisible && descriptor.m_imguiComparisonWindowOpened)
```

Comment:
Why was this code changed? Was the merge conflict incorrectly resolved?

Reply:
Yes, I put it back